src/Calypso-SystemQueries/Project.extension.st

```diff
@@ -1,7 +1,7 @@
 Extension { #name : 'Project' }
 
 { #category : '*Calypso-SystemQueries' }
-Project class >> convertToCalypsoBrowserItem: aProject [
```

Comment:
I got this change because of a merge. @jecisc do you know why?

Reply:
I have no idea where those "classSide" are coming from.. I've see it in the past in some projects but I never discovered why. 

I never do my merges with Pharo, maybe it's the Pharo merger that uses another printer?